app/views/directives/edit-environment-from.html

```diff
@@ -4,10 +4,35 @@
     class="environment-from-entry environment-from-editor-entry-header">
     <div class="form-group environment-from-editor-header value-header">
       <div class="input-group">
-        <span class="help-block">{{$ctrl.selectorPlaceholder}}</span>
+        {{$ctrl.selectorPlaceholder}}
+      </div>
+    </div>
+    <div class="form-group environment-from-editor-header value-header">
+      <div class="input-group">
+        Prefix
+        <small class="pficon pficon-help"
+               aria-hidden="true"
+               data-toggle="tooltip"
+               data-original-title="Optional prefix added to each environment variable name for this resource."></small>
```

Comment:
Maybe remove "for this resource." since it's a column header with many resources.

I know I'm picking nits on my own suggested wording :)

Reply:
agree, drop the "for this resource"